packages/mobile/dapp-store/pnpm-lock.yaml

```diff
@@ -0,0 +1,2360 @@
+lockfileVersion: 5.4
```

Comment:
hey just wondering what this file is, is this the new npm 16 thing?

Reply:
they use pnpm 🤷 
https://github.com/solana-mobile/app-publishing-spec/blob/main/packages/cli/README.md